src/docdb/commands.ts

```diff
@@ -65,14 +66,20 @@ export class DocDBCommands {
     }
 
 
-    public static async createDocDBCollection(db: DocDBDatabaseNode, explorer: CosmosDBExplorer) {
+    public static async createCollection(db: DocDBDatabaseNode | GraphDatabaseNode, explorer: CosmosDBExplorer) {
+        let placeHolder: string;
+        if (db instanceof GraphDatabaseNode) {
```

Comment:
Shouldn't this be the other way around? 
If the db is a GraphDatabaseNode I ask for name of a graph, 
else I ask for name of a collection.